### ISSUE TITLE
Cannot generate more than once without reopening the Gradio UI

### ISSUE THREAD

**Issue by herbertwestern** [TheLastBen/fast-stable-diffusion#806]
The generate button will not respond to presses after the first batch of images is done. There is no error message, but clicking on it does nothing. Reopening the Gradio window by clicking the link again allows one more generation, etc... Basically, I need to reopen via the link for each batch.

**Comment by dravid-cmyk** [comment]
same error @TheLastBen  please have a look

**Comment by BaptisteGarcin** [comment]
same

**Comment by konqiDAM** [comment]
yeah same, so anying it hangs, one time the trick of shift + intro works, not more

**Comment by TheLastBen** [comment]
trying to reproduce the error but I can't, it seems that the server is unstable

**Comment by dravid-cmyk** [comment]
is server stable now @BaptisteGarcin @konqiDAM @TheLastBen @herbertwestern

**Comment by herbertwestern** [comment]
@dravid-cmyk 

I won't have time to try today, unfortunately.

@TheLastBen 

If it helps, I've been using it on Chrome in Desktop Mode on Android 11. Not sure if that's at all relevant, but I thought I'd mention.

**Comment by herbertwestern** [comment]
@TheLastBen , this issue is back after the latest Automatic 1111 update.

**Comment by herbertwestern** [comment]
> nothing was changed in the gradio area

@TheLastBen Unsure of what the issue could be then, but it is indeed happening. Sorry I can't help more! Just the messenger.

**Comment by TheLastBen** [comment]
Sometimes the server lags and causes gradio to freeze as it is very sensitive to that.

**Comment by herbertwestern** [comment]
> Sometimes the server lags and causes gradio to freeze as it is very sensitive to that.

Gotcha! Yeah, I'm in the position of not completely understanding the nature of these issues, so I hope it doesn't come across the wrong way, like, "fix it!" Not my intention--just trying to report what's happening and it's understood if it's out of your control. I just don't have the technical insight to know that beforehand. Thanks for all the work put into this!
